web3_proxy/src/app/mod.rs

```diff
@@ -167,11 +170,12 @@ pub struct AuthorizationChecks {
 
 /// Simple wrapper so that we can keep track of read only connections.
 /// This does no blocking of writing in the compiler!
-#[derive(Clone)]
+#[derive(Clone, Debug)]
 pub struct DatabaseReplica(pub DatabaseConnection);
 
 // TODO: I feel like we could do something smart with DeRef or AsRef or Borrow, but that wasn't working for me
 impl DatabaseReplica {
+    #[instrument(skip_all)]
```

Comment:
is it worth adding instrument to functions that don't log anything? is that necessary to make sure spans carry over properly? I'm honestly not sure.